docs/ide/finding-and-replacing-text.md

```diff
@@ -116,9 +116,9 @@ Use the following keys and actions for multi-caret selection behaviors:
 |**Ctrl**+**Alt** + double-click|Add a secondary word selection|
 |**Ctrl**+**Alt** + click + drag|Add a secondary selection|
 |**Shift**+**Alt**+**.**|Add the next matching text as a selection|
-|**Ctrl**+**Shift**+**Alt**+**,**|Add all matching text as selections|
+Updated shortcuts |**Shift**+**Alt**+**;**|Add all matching text as selections|
```

Comment:
@DavidJohnWilliams This is great; thanks ... but, please remove the text that says "Updated shortcuts" - and instead, update the ms.date value in the metadata with the current date. Thanks. :)